changes/changes-2.4.0.md

```diff
@@ -0,0 +1,12 @@
+Changes by Version
+==================
+Release Notes.
+
+Apollo Java 2.4.0
+
+------------------
+* [Feature Support Kubernetes ConfigMap cache for Apollo java, golang client](https://github.com/apolloconfig/apollo-java/pull/79)
```

Comment:
There is no need to update the `changes/changes-2.4.0.md`, as it will be updated after the release of 2.4.0.